Makefile

```diff
@@ -25,14 +25,14 @@ refman-release:
 	make -C ${DIR}/docs/DafnyRef release
 
 z3-mac:
-	wget https://github.com/Z3Prover/z3/releases/download/z3-4.8.4/z3-4.8.4.d6df51951f4c-x64-osx-10.14.1.zip
-	unzip z3-4.8.4.d6df51951f4c-x64-osx-10.14.1.zip
-	mv z3-4.8.4.d6df51951f4c-x64-osx-10.14.1 ${DIR}/Binaries/z3
+	wget https://github.com/Z3Prover/z3/releases/download/Z3-4.8.5/z3-4.8.5-x64-osx-10.14.2.zip
```

Comment:
Since we're updating Z3, should we try to update it to the latest version? `z3-4.8.9-x64-osx-10.14.6.zip`?

Reply:
Additional note. If we update this here, we should also update the `INSTALL.md` Z3 to match: https://github.com/dafny-lang/dafny/blob/master/INSTALL.md

Reply:
I've tried to update to Z3 v.4.8.9, but too many tests are failing (and the same with v4.8.8 and the nightly build of what will become v4.8.10). However, the switch from v4.8.4 to v4.8.5 still seemed worthwhile, because there was a renamed option in v4.8.5 that the new Boogie relies on.

Reply:
I updated the INSTALL.md file